### PR TITLE
feat: add contentbusid surrogate key for pipeline responses depending…

### PR DESCRIPTION
**Pull request by stefan-guggisberg**
… on content resources

Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Review by tripodsan — APPROVED** [review]

**Review by dominique-pfister — APPROVED** [review]

File: src/steps/fetch-content.js [hunk 1]
```diff
@@ -41,7 +41,15 @@ export default async function fetchContent(state, req, res) {
       redirectLocation += '.plain.html';
     }
     res.headers.set('location', redirectLocation);
-    res.headers.set('x-surrogate-key', await computeSurrogateKey(`${contentBusId}${info.path}`));
```
Comment (stefan-guggisberg) [inline src/steps/fetch-content.js:44]: @tripodsan is the following correct?

File: src/json-pipe.js [hunk 2]
```diff
@@ -61,7 +61,12 @@ async function fetchJsonContent(state, req, res) {
     res.body = '';
     res.headers.delete('content-type');
     res.headers.set('location', redirectLocation);
-    res.headers.set('x-surrogate-key', await computeSurrogateKey(`${contentBusId}${info.path}`));
+    const keys = [];
+    keys.push(await computeSurrogateKey(`${contentBusId}${info.path}`));
+    if (state.content.sourceBus === 'content') {
+      keys.push(contentBusId);
+    }
```
Comment (tripodsan) [inline src/json-pipe.js:68]: why not the `_code` one ?

File: src/steps/fetch-content.js (hunk 1 above)
Comment (tripodsan) [reply src/steps/fetch-content.js:44]: I think so :-)

File: src/json-pipe.js (hunk 2 above)
Comment (stefan-guggisberg) [reply src/json-pipe.js:68]: good catch!

**Review by tripodsan — APPROVED** [review]

**Pull request merged**